.all-contributorsrc

```diff
@@ -0,0 +1,28 @@
+{
+  "projectName":"react-router-docs",
+  "projectOwner":"knowbody",
+  "files":[
+    "README.md"
+  ],
+  "imageSize":100,
+  "contributors":[
```

Comment:
Yikes, the formatting is messed up. No spaces between property names and values. Let me fix that